app/models/item.rb

```diff
@@ -12,6 +12,7 @@ def self.default_scope
   has_many :bin_items
   has_many :bins, -> { includes(:bin_location).order(:label) }, through: :bin_items
   validates :description, presence: true
+  validates :value, numericality: { other_than: 0.0 }
```

Comment:
Do we want to restrict this to positive values as well?

Reply:
not for now